fieldservice_account_analytic/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    filter_location_by_contact = fields.Boolean(
```

Comment:
@osi-scampbell I would prefix the field with `fsm_` as we could have the same with stock...